pkg/vpcmodel/explainabilityInput.go

```diff
@@ -224,21 +212,34 @@ func (c *VPCConfig) getNodesFromInputString(cidrOrName string) (nodes []Node, in
 			fmt.Errorf("%s %s", cidrOrName, noValidInputMsg)
 	}
 	// the input is a legal cidr or IP address
-	return c.getNodesFromAddress(suffixNameOrCidr, ipBlock)
+	return c.getNodesFromAddress(cidrOrName, ipBlock)
 }
 
 // getNodesOfVsi gets a string name or UID of VSI, and
 // returns the list of all nodes within this vsi
-func (c *VPCConfig) getNodesOfVsi(vpc, vsi string) ([]Node, int, error) {
+func (c *VPCConfig) getNodesOfVsi(name string) ([]Node, int, error) {
 	var nodeSetWithVsi NodeSet
+	// vsi name may be prefixed by vpc name
+	var vpc, vsi string
+	uid := name // uid specified - vpc prefix is not relevant and uid may contain "/"
+	cidrOrNameSlice := strings.Split(name, "/")
+	switch len(cidrOrNameSlice) {
+	case 1: // vpc name not specified
+		vsi = name
+	case 2: // vpc name specified
+		vpc = cidrOrNameSlice[0]
+		vsi = cidrOrNameSlice[1]
+	default: // > 1, address
+		vsi = name
+	}
```

Comment:
why do you differentiate case `1` and `default` here? 
where is `address` assumed?

Reply:
the address is from the time I used "."
No longer relevant, removed.

Reply:
shouldn't issue an error in this case?

Reply:
no. uid may contain more than one such deliminator since its a `crn`. btw, I still don't see why `UID` is `crn` and not `id`.